arts-main/wasm_files/file_handler.py

```diff
@@ -0,0 +1,23 @@
+import os
+import base64
+
+from django.conf import settings
+from pubsub import messages
+
+def get_file_path(instance, filename):
+    """ Return unix style file path. 
```

Comment:
Docstrings should use [this format](https://numpydoc.readthedocs.io/en/latest/format.html) in order to be read correctly by the autodoc.